libs/markets/src/lib/components/market-info/info-key-value-table.tsx

```diff
@@ -22,17 +24,26 @@ interface RowProps {
   unformatted?: boolean;
   assetSymbol?: string;
   noBorder?: boolean;
+  parentValue?: ReactNode;
+  hasParentData?: boolean;
 }
 
-const Row = ({
+export const Row = ({
   field,
   value,
   decimalPlaces,
   asPercentage,
   unformatted,
   assetSymbol = '',
   noBorder = true,
+  parentValue,
+  hasParentData,
 }: RowProps) => {
+  // Note: we need both 'parentValue' and 'hasParentData' to do a conditional
+  // check to differentiate between when parentData itself is missing and when
+  // a specific parentValue is missing. These values are only used when we
+  // have successor market parent data.
```

Comment:
**praise. (non-blocking)**: Nice useful commenting. Soundssimple, but something you could lose a lot of time to.

Reply:
thanks